ambari/src/main/java/io/brooklyn/ambari/server/AmbariServerImpl.java

```diff
@@ -137,30 +136,7 @@ protected void connectSensors() {
                 .computing(Functionals.ifNotEquals(true).value("URL not reachable"))
                 .build());
 
-        hostsHttpFeed = HttpFeed.builder()
-                .entity(this)
-                .period(1000, TimeUnit.MILLISECONDS)
-                .baseUri(String.format("%s/api/v1/hosts", ambariUri))
-                .credentials(usernamePasswordCredentials.getUserName(), usernamePasswordCredentials.getPassword())
-                .header(HttpHeaders.AUTHORIZATION, HttpTool.toBasicAuthorizationValue(usernamePasswordCredentials))
-                .poll(new HttpPollConfig<List<String>>(REGISTERED_HOSTS)
-                        .onSuccess(Functionals.chain(HttpValueFunctions.jsonContents(), getHosts()))
-                        .onFailureOrException(Functions.<List<String>>constant(ImmutableList.<String>of())))
-                .build();
-
-        clusterHttpFeed = HttpFeed.builder()
-                .entity(this)
-                .period(1000, TimeUnit.MILLISECONDS)
-                .baseUri(String.format("%s/api/v1/clusters/%s/requests/%d",
-                        ambariUri,
-                        "Cluster1",
-                        1))
-                .credentials(usernamePasswordCredentials.getUserName(), usernamePasswordCredentials.getPassword())
-                .header(HttpHeaders.AUTHORIZATION, HttpTool.toBasicAuthorizationValue(usernamePasswordCredentials))
-                .poll(new HttpPollConfig<String>(CLUSTER_STATE)
-                        .onSuccess(Functionals.chain(HttpValueFunctions.jsonContents(), getRequestState()))
-                        .onFailureOrException(Functions.<String>constant(null)))
-                .build();
+        connectSensorsHelper();
```

Comment:
Is there a better way to name this - possibly connectAuthenticatedSensors? - just to try to make it obvious why it is separate from the rest.